src/components/MessagesList/MessagesGroup/Message/MessagePart/PollVotersDetails.vue

```diff
@@ -20,7 +20,8 @@
 -->
 
 <template>
-	<NcPopover trigger="hover">
+	<NcPopover trigger="hover"
+		:focus-trap="false">
```

Comment:
this is very cumbersome.... After "Show results" I have to tab myself through all the remaining UI including the participant listing before I reach the popup to find the poll voting details

Reply:
But that seems also the case on master, is it not this?
Is the container missing on the Poll Modal?

Reply:
I can't reproduce this neither on chrome nor firefox. As soon as i hit enter on see result I'm in the dialog with the focus on the "pollVotersDetails" component. Could make sure you pulled and test again pleez?

Reply:
Hmm seems to work on this PR.

After rebasing to master (was lacking 100 commits behind) it does work as I described. Might be the broken focus trap thing in 7.0.1+

So I'd say let's wait for a new vue lib version and then revisit the PR?

Reply:
ok sounds good!